migration/v2.16.0-ck8s1-v2.17.0-ck8s1/upgrade-cluster.md

```diff
@@ -0,0 +1,8 @@
+# Upgrade v2.16.0-ck8s1 to v2.17.0-ck8s1
+
+1. Checkout the new release: `git checkout v2.17.0-ck8s1`
+
+1. Update the kubespray submodule: `git submodule update --init --recursive`
+
+1. Upgrade your cluster by running `./bin/ck8s-kubespray run-playbook sc upgrade-cluster.yml -b`.
+    Note that this will also upgrade Kubernetes to v1.21.5 (unless you have another version pinned).
```

Comment:
```suggestion
    Note that this will also upgrade Kubernetes to v1.21.5 (unless you have another version pinned).
    
1. Upgrade your cluster by running `./bin/ck8s-kubespray run-playbook wc upgrade-cluster.yml -b`.
    Note that this will also upgrade Kubernetes to v1.21.5 (unless you have another version pinned).
```